.github/workflows/ci-javascript.yaml

```diff
@@ -204,18 +172,11 @@ jobs:
       - name: Build
         run: yarn build
 
-      - name: Get updated wrappers
-        if: ${{ needs.CI-WRAP-Test-Harness.outputs.rebuild_required == 'true' }}
-        uses: actions/download-artifact@v3
-        id: get-wrappers
-        with:
-          name: rebuilt-wrappers
-          path: ./wrappers
+      - name: Test cli:unit
+        run: yarn test:cli:unit
 
-      - if: ${{ needs.CI-WRAP-Test-Harness.outputs.rebuild_required == 'true' }}
-        run: |
-          rm -rf packages/test-cases/cases/wrappers
-          mv wrappers packages/test-cases/cases
+      - name: Test cli:e2e:p1
```

Comment:
just curious: what `p1` and `p2` mean?

Reply:
"Part 1" and "Part 2". I wanted to split up the CLI tests to narrow in on what tests were failing, since there wasn't any output being emitted in CI.